hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/AbstractS3AMockTest.java

```diff
@@ -80,6 +80,15 @@ public Configuration createConfiguration() {
     conf.setInt(ASYNC_DRAIN_THRESHOLD, Integer.MAX_VALUE);
     // set the region to avoid the getBucketLocation on FS init.
     conf.set(AWS_REGION, "eu-west-1");
+
+    // tight retry logic as all failures are simulated
+    final String interval = "1ms";
+    final int limit = 3;
```

Comment:
Can this change to 2?

Reply:
can do, but it's nice and fast anyway

Reply:
actually, I'd rather not. because this test is shared with other mock tests and it is probably a bit less risky to cut back on all of them just to speed up one test